api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -181,8 +183,9 @@ public GoogleCredential getDefaultServiceAccountCredentials() throws IOException
   }
 
   @Override
-  public JSONObject getFileAsJson(String bucketName, String fileName) throws IOException {
-    return new JSONObject(readToString(bucketName, fileName));
+  public JsonObject getFileAsJson(String bucketName, String fileName) throws IOException {
```

Comment:
This change is necessary because of the incompatibility between `org.json.JSONObject` and `com.google.gson.JsonObject`.  We should fix this: https://precisionmedicineinitiative.atlassian.net/browse/RW-2929